src/sonic-host-services/setup.py

```diff
@@ -1,5 +1,11 @@
 from setuptools import setup
 
+dependencies = [
+    'natsort',
+    'sonic_py_common',
+    'swsssdk>=2.0.1',
+]
```

Comment:
`dependencies` is not being used. Please remove this and add these dependencies to the `install_requires` list below instead. Also, it doesn't look like `natsort` is being used and the other two dependencies are already there.